Test/testFileSpecfic.toml

```diff
@@ -0,0 +1,40 @@
\tsubstitution\tN->H\t\tsubstitution\tY->W\t\tsubstitution\tI->H\t\tsubstitution\tL->H\t\tsubstitution\tC->Q\t\tsubstitution\tM->R\t\tsubstitution\tC->K\t\tsubstitution\tH->Y\t\tsubstitution\tA->P\t\tsubstitution\tC->E\t\tsubstitution\tS->L or I\t\tsubstitution\tS->N\t\tsubstitution\tT->Q\t\tsubstitution\tT->K\t\tsubstitution\tE->R\t\tsubstitution\tS->D\t\tsubstitution\tT->E\t\tsubstitution\tK->R\t\tsubstitution\tA->V\t\tsubstitution\tC->M\t\tsubstitution\tQ->R\t\tsubstitution\tV->Q\t\tsubstitution\tV->K\t\tsubstitution\tV->E\t\tsubstitution\tR->W\t\tsubstitution\tT->M\t\tsubstitution\tA->T\t\tsubstitution\tG->S\t\tsubstitution\tP->Q\t\tsubstitution\tP->K\t\tsubstitution\tA->C\t\tsubstitution\tV->M\t\tsubstitution\tM->Y\t\tsubstitution\tD->F\t\tsubstitution\tN->F\t\tsubstitution\tI->F\t\tsubstitution\tL->F\t\tsubstitution\tP->M\t\tsubstitution\tE->Y\t\tsubstitution\tC->H\t\tsubstitution\tK->Y\t\tsubstitution\tQ->Y\t\tsubstitution\tU->W\t\tsubstitution\tD->U\t\tsubstitution\tT->H\t\tsubstitution\tN->U\t\tsubstitution\tI->U\t\tsubstitution\tL->U\t\tsubstitution\tV->H\t\tsubstitution\tF->W\t\tsubstitution\tP->H\t\tsubstitution\tG->P\t\tsubstitution\tS->Q\t\tsubstitution\tS->K\t\tsubstitution\tD->R\t\tsubstitution\tS->E\t\tsubstitution\tA->L or I\t\tsubstitution\tG->V\t\tsubstitution\tN->R\t\tsubstitution\tA->N\t\tsubstitution\tI->R\t\tsubstitution\tL->R\t\tsubstitution\tA->D\t\tsubstitution\tS->M\t\tsubstitution\tG->T\t\tsubstitution\tC->F\t\tsubstitution\tG->C\t\tsubstitution\tT->F\t\tsubstitution\tC->U\t\tsubstitution\tV->F\t\tsubstitution\tD->Y\t\tsubstitution\tH->W\t\tsubstitution\tN->Y\t\tsubstitution\tT->U\t\tsubstitution\tI->Y\t\tsubstitution\tL->Y\t\tsubstitution\tP->F\t\tsubstitution\tS->H\t\tsubstitution\tV->U\t\tsubstitution\tC->R\t\tsubstitution\tP->U\t\tsubstitution\tM->W\t\tsubstitution\tT->R\t\tsubstitution\tG->L or I\t\tsubstitution\tV->R\t\tsubstitution\tE->W\t\tsubstitution\tA->K\t\tsubstitution\tK->W\t\tsubstitution\tA->E\t\tsubstitution\tG->D\t\tsubstitution\tQ->W\t\tsubstitution\tP->R\t\tsubstitution\tA->M\t\tsubstitution\tS->F\t\tsubstitution\tC->Y\t\tsubstitution\tT->Y\t\tsubstitution\tS->U\t\tsubstitution\tV->Y\t\tsubstitution\tP->Y\t\tsubstitution\tA->H\t\tsubstitution\tS->R\t\tsubstitution\tG->Q\t\tsubstitution\tD->W\t\tsubstitution\tG->K\t\tsubstitution\tG->E\t\tsubstitution\tN->W\t\tsubstitution\tI->W\t\tsubstitution\tL->W\t\tsubstitution\tG->M\t\tsubstitution\tS->Y\t\tsubstitution\tA->F\t\tsubstitution\tA->U\t\tsubstitution\tG->H\t\tsubstitution\tC->W\t\tsubstitution\tT->W\t\tsubstitution\tA->R\t\tsubstitution\tV->W\t\tsubstitution\tP->W\t\tsubstitution\tG->F\t\tsubstitution\tA->Y\t\tsubstitution\tG->U\t\tsubstitution\tS->W\t\tsubstitution\tG->R\t\tsubstitution\tG->Y\t\tsubstitution\tA->W\t\tsubstitution\tG->W\t\tSurfactant\tHydrogen Tetradecyl Sulfate\t\tSurfactant\tHydrogen Dodecyl Sulfate\t\tTandemMassTag\tTMT_tag_lysine\t\tTandemMassTag\tTMT_tag_terminal\t\tConnorModType\tConnorMod\t\ttestUniqueModType\ttestUniqeMod"
```

Comment:
Just make the ListOfModsLocalize  list empty. It should be taken care of by LocalizeAll = true